hnclient-v2/HNClient-Service.xml

```diff
@@ -0,0 +1,26 @@
+
+
+<!--
+ This is an example of a HNCLient Windows Service Wrapper configuration, which includes only mandatory options.
+ 
+ This configuration file should be placed near the WinSW executable, the name should be the same.
+ E.g. for HNClient-Service.exe the configuration file name should be  HNClient-Service.xml
+ 
+ You can find more information about the configuration options here: https://github.com/winsw/winsw/releases/tag/v2.11.0 sample-minimal.xml
+ and sample-allOptions.xml
+ 
+-->
+<configuration>
+  
+  <!-- ID of the service. It should be unique across the Windows system-->
+  <id>hnclient-service-new</id>
+  <!-- Display name of the service -->
+  <name>HNClient App Service</name>
```

Comment:
I think in general for documentation and artifacts we want to refer to HNS Client. I believe Trevor said that will be the official name.